docs/configuration.rst

```diff
@@ -190,6 +190,40 @@ below:
 
          **Note** that the JSON object must be included in single quotes (') to avoid problems with escaping of the double quote (") character.
 
+   DD_SPAN_SAMPLING_RULES:
+     type: string
+     description: |
+         A JSON array of objects. Each object must have a “name” and/or “service” field, while the "max_per_second" and “sample_rate” fields are optional. 
+         The “sample_rate” value must be between 0.0 and 1.0 (inclusive), and will default to 1.0 (100% sampled).
+         The "max_per_second" value must be >= 0 and will default to no limit. 
+         The "service" and "name" fields are glob patterns, where "glob" means: 
+         "*" matches any substring, including the empty string,
+         "?" matches exactly one of any character, and
+         any other character matches exactly one of itself.
```

Comment:
any reason for the newline here? not sure how this formats

Reply:
It's just what we do with `DD_TRACE_SAMPLING_RULES` so I thought it would be best to follow the same formatting. It looks like this which I think is clear?: https://ddtrace.readthedocs.io/en/latest/configuration.html#DD_TRACE_SAMPLING_RULES